src/Features/CSharp/Portable/IntroduceVariable/CSharpIntroduceVariableService_IntroduceLocal.cs

```diff
@@ -47,14 +50,23 @@ protected override Document IntroduceLocal(
             ? TokenList(ConstKeyword)
             : default;
 
+        var updatedExpression = expression.WithoutTrivia();
+
+        if (options.SimplifierOptions is CSharpSimplifierOptions csOptions && csOptions.ImplicitObjectCreationWhenTypeIsApparent.Value
+            && csOptions.GetUseVarPreference() == UseVarPreference.None)
+        {
+            if (expression is ObjectCreationExpressionSyntax oce)
```

Comment:
we don't use single letter abbreviations.  this would be `objectCreationExpression`